### DISCUSSION TITLE
"no such file or directory" error with Git Storage

### DISCUSSION THREAD

**Discussion by jackytsu** [requarks/wiki#4144] [Error / Bug Report] [answered]
First, I edited md file, and pushed to the sever.
Then wikijs server runs "Force Sync", I got an "no such file or directory" error in log. Of course, the modifications of the md file are not presented.
But, if I run "Import Everything", I can see the modified content on the web.

Log screenshot of "Force Sync"
![image](https://user-images.githubusercontent.com/608014/123756858-ef63a380-d8ef-11eb-8547-3ec93eb49c2b.png)

Log screenshot of "Import Everything"
![image](https://user-images.githubusercontent.com/608014/123757376-7dd82500-d8f0-11eb-9133-c94e1ce3f806.png)

**Comment by EricFromCanada** [comment, marked as answer]
Fixed in https://github.com/requarks/wiki/pull/5272
